gb-ppu/src/registers/lcd/palettes_mono.rs

```diff
@@ -1,27 +1,47 @@
-use super::super::Palette;
+use super::super::{LcdReg, Palette};
```

Comment:
The name of the file could be changed since it is no longer only about mono palettes